reference/config_files/profiles.rst

```diff
@@ -0,0 +1,519 @@
+**environment variables** (for build time and runtime context), **tool requirements**, and
+**configuration variables** in a file.
+
+They have this structure:
+
+.. code-block:: text
+
+    [settings]
+    arch=x86_64
+    build_type=Release
+    os=Macos
+
+    [options]
+    MyLib:shared=True
+
+    [buildenv]
+    VAR1=value
+
+    [tool_requires]
+    tool1/0.1@user/channel
+    *: tool4/0.1@user/channel
+
+    [conf]
+    tools.build:jobs=2
+
+
+Profiles can be created with ``detect`` option in :ref:`conan profile <reference_commands_profile>` command,
+and edited later. If you don't specify a *name*, the command will create the ``default`` profile:
+
+
+.. code-block:: bash
+    :caption: *Creating the Conan default profile*
+
+    $ conan profile detect
+    Found apple-clang 12.0
+    Detected profile:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+
+    WARN: This profile is a guess of your environment, please check it.
+    WARN: The output of this command is not guaranteed to be stable and can change in future Conan versions
+    Saving detected profile to [CONAN_HOME]/profiles/default
+
+.. code-block:: bash
+    :caption: *Creating another profile: myprofile*
+
+    $ conan profile detect --name myprofile
+    Found apple-clang 12.0
+    Detected profile:
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+
+    WARN: This profile is a guess of your environment, please check it.
+    WARN: The output of this command is not guaranteed to be stable and can change in future Conan versions
+    Saving detected profile to [CONAN_HOME]/profiles/myprofile
+
+
+Profile files can be used with ``-pr``/``--profile`` option in many commands like :command:`conan install` or
+:command:`conan create` commands. If you don't specify any profile at all, the profile ``default`` will be
+always used:
+
+.. code-block:: bash
+    :caption: Using the *default* profile
+
+    $ conan create .
+
+
+.. code-block:: bash
+    :caption: Using a *myprofile* profile
+
+    $ conan create . -pr=myprofile
+
+
+Profiles can be located in different folders:
+
+.. code-block:: bash
+
+    $ conan install . -pr /abs/path/to/myprofile   # abs path
+    $ conan install . -pr ./relpath/to/myprofile   # resolved to current dir
+    $ conan install . -pr ../relpath/to/myprofile  # resolved to relative dir
+    $ conan install . -pr myprofile  # resolved to [CONAN_HOME]/profiles/myprofile
+
+Listing existing profiles in the *profiles* folder can be done like this:
+
+.. code-block:: bash
+
+    $ conan profile list
+    Profiles found in the cache:
+    default
+    myprofile1
+    myprofile2
+    ...
+
+You can also show profile's content:
```

Comment:
```suggestion
You can also show profile's contents:
```

Reply:
Actually, it's only one content. It shows several contexts but the content is exactly the same for both.
Perhaps, something like:

```suggestion
You can also show the profile's content per context:
```
Wdyt?